pages/learn/getting-started-step-by-step.md

```diff
@@ -176,43 +174,40 @@ To define a required property:
 
 1. Inside the `properties` object, add the `price` key. Include the usual schema annotations `description` and `type`, where `type` is a number:
 
-
-```jsonc
-  "properties": {
-    ...
-    "price": {
-      "description": "The price of the product",
-      "type": "number"
-    }
-  }
+    ```jsonc
+      "properties": {
+        ...
+        "price": {
+          "description": "The price of the product",
+          "type": "number"
+        }
+      }
 ```
```

Comment:
this one missed indentation

Reply:
Thanks for finding this @notEthan !! 

@bhavukkalra can you create a new PR to fix this? I merged it too fast.